selfdrive/debug/print_docs_diff.py

```diff
@@ -1,117 +1,123 @@
+  # experimental mode
+  exp_link = "<a href='https://blog.comma.ai/090release/#experimental-mode' target='_blank' class='link-light-new-regular-text'>Experimental mode</a>"
+  if openpilotLongitudinalControl and not experimentalLongitudinalAvailable:
+    sentence_builder += f" Traffic light and stop sign handling is also available in {exp_link}."
+  return sentence_builder.format(car_model=f"{make} {model}", alc=alc, acc=acc)
+
+def process_detail_sentences(info):
+  detail_sentences = []
+  ind = info.index("---")
+  for line1, line2 in zip(info[1:ind], info[ind+1:], strict=True):
+    name = ' '.join(line1[:2].split("|")[:2])
+    detail1 = get_detail_sentence(line1[2:])
+    detail2 = get_detail_sentence(line2[2:])
+    if detail1 != detail2:
+      detail_sentences.append(f"- Sentence for {name} changed!\n" +
                                  "  ```diff\n" +
-                                 f"  - {base_car.detail_sentence}\n" +
-                                 f"  + {new_car.detail_sentence}\n" +
+                                 f"  - {detail1}\n" +
+                                 f"  + {detail2}\n" +
                                  "  ```")
+  return detail_sentences
+
+def process_diff_information(info):
+  header = info[0]
+  category = None
+  final_strings = []
+  if "c" in header:
+    category = "column"
+    ind = info.index("---")
+    for line1, line2 in zip(info[1:ind], info[ind+1:], strict=True):
+      final_strings.append(column_change_format(line1[2:], line2[2:]))
+  else:
+    category = "additions" if "a" in header else "removals"
+    final_strings = [x[2:] for x in info[1:]]
+  return category, final_strings
+
+def print_markdown(changes):
+  markdown_builder = ["### ⚠️ This PR makes changes to [CARS.md](../blob/master/docs/CARS.md) ⚠️"]
+  for title, category in (("## 🔀 Column Changes", "column"), ("## ❌ Removed", "removals"),
+                          ("## ➕ Added", "additions"), ("## 📖 Detail Sentence Changes", "detail")):
+    # TODO: Add details for detail changes
+    if len(changes[category]):
+      markdown_builder.append(title)
+      if category not in ("detail",):
+        markdown_builder.append(COLUMNS)
+        markdown_builder.append(COLUMN_HEADER)
+      markdown_builder.extend(changes[category])
+  print("\n".join(markdown_builder))
 
-  # Print diff
-  if any(len(c) for c in changes.values()):
-    markdown_builder = ["### ⚠️ This PR makes changes to [CARS.md](../blob/master/docs/CARS.md) ⚠️"]
-
-    for title, category in (("## 🔀 Column Changes", "column"), ("## ❌ Removed", "removals"),
-                            ("## ➕ Added", "additions"), ("## 📖 Detail Sentence Changes", "detail")):
-      if len(changes[category]):
-        markdown_builder.append(title)
-        if category not in ("detail",):
-          markdown_builder.append(COLUMNS)
-          markdown_builder.append(COLUMN_HEADER)
-        markdown_builder.extend(changes[category])
+def print_car_docs_diff(path):
```

Comment:
path should be a path to the base temporary CARS.md file downloaded in the Action, and use BASEDIR to find the current branch CARS.md, which is always in a static location, so that should be a global constant:

`CARS_MD_OUT = os.path.join(BASEDIR, "docs", "CARS.md")`